### ISSUE TITLE
References to a Failure for 2.5.2 Pointer Cancellation is out of date

### ISSUE THREAD

**Issue by goodwitch** [w3c/wcag#372]
Dear AGWG & specifically the Mobile Task Force, 

I was just looking closely at the understanding document for WCAG 2.1 SC 2.5.2 Pointer Cancellation...and I've got a question for you.

If you go to [Understanding SC 2.5.2 Pointer Cancellation](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html) and find the link text that reads 

[FM001 Failure of SC 2.5.3 due to activating a button on initial touch location rather than the final touch location](http://w3c.github.io/Mobile-A11y-TF-Note/Techniques/FM001)
And you click on that link, it goes to a page that has details about a different failure technique. The H1 of that linked page is "Content or functionality not available by touch gesture when platform assistive technology is active"

Something is out of sync.  I had a brief conversation with Jon Avila about it, and he suggested that this was an old technique that has morphed into something different, but the old wording in the Failure Technique was never updated.  He suggested I submit an issue :)

**Comment by alastc** [comment]
It links through to a wiki page, which needs to be a temporary situation, we should get a technique into the repo for that.

**Comment by awkawk** [comment]
I propose that unless we resolve this at the F2F that we remove this reference until it actually exists.
